src/components/playback/playbackmanager.js

```diff
@@ -2385,8 +2388,18 @@ class PlaybackManager {
                 playOptions.items = null;
 
                 return getPlaybackMediaSource(player, apiClient, deviceProfile, maxBitrate, item, startPosition, mediaSourceId, audioStreamIndex, subtitleStreamIndex).then(function (mediaSource) {
-                    if (userSettings.enableSetUsingLastTracks())
-                        autoSetNextTracks(prevSource, mediaSource);
+                    const user = apiClient.getCurrentUser();
```

Comment:
`getCurrentUser` returns Promise

Add `async` above:
```
                return getPlaybackMediaSource(player, apiClient, deviceProfile, maxBitrate, item, startPosition, mediaSourceId, audioStreamIndex, subtitleStreamIndex).then(async (mediaSource) => {
```

```suggestion
                    const user = await apiClient.getCurrentUser();
```